models.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"time"
+)
+
+// request and response structures
+
+// Response structure for /status call.
+type StatusResponse struct {
+	Task_id   string     `json:"task_id"`
+	Finished  *bool      `json:"finished,omitempty"`
+	StartTime *time.Time `json:"start_time,omitempty"`
+	EndTime   *time.Time `json:"end_time,omitempty"`
+	ExitCode  *int       `json:"exit_code,omitempty"`
+	Output    []string   `json:"output,omitempty"`
+	Errors    []string   `json:"errors,omitempty"`
```

Comment:
It would be helpful if these attributes were documented.
Also, why errors as a slice not a a single error - where is it set, what kind of errors are these and how is client supposed to handle them?

Reply:
I changed it back to a *string. I realized if the original call returned with an error, the StartCommand handler would not have executed the subroutine.